backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/webservice/FreinetApplicationMutationService.kt

```diff
@@ -57,21 +57,30 @@ class FreinetApplicationMutationService {
             val lastName = personalDataNode?.get("value")?.findValueByName("surname").orEmpty()
             val dateOfBirth = personalDataNode?.get("value")?.findValueByName("dateOfBirth").orEmpty()
 
-            val persons = freinetSearchPersonApi.searchPersons(
-                firstName = firstName,
-                lastName = lastName,
-                dateOfBirth = dateOfBirth,
-                accessKey = freinetAgency.apiAccessKey,
-                agencyId = freinetAgency.agencyId,
+            val freinetApi = FreinetApi(projectConfig.freinet.host, freinetAgency.apiAccessKey, freinetAgency.agencyId)
+
+            val persons = freinetApi.searchPersons(
+                firstName,
+                lastName,
+                dateOfBirth,
             )
             when {
                 persons.size() > 1 -> {
                     logger.warn("Multiple persons found in Freinet for $firstName $lastName, born $dateOfBirth")
                     throw FreinetFoundMultiplePersonsException()
                 }
                 persons.isEmpty() -> {
-                    // TODO: #2142 - Create person for Freinet
-                    logger.warn("create person for freinet")
+                    val personCreationResult = freinetApi.createPerson(
```

Comment:
🔧 I think you don't need a result here, if you throw the exception directly in the `createPerson` fkt